frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -631,12 +672,20 @@ export const ClusterServiceVersionsPage: React.FC<ClusterServiceVersionsPageProp
     </p>
   );
 
-  const flatten = ({ clusterServiceVersions, subscriptions }) =>
-    [
-      ..._.get(clusterServiceVersions, 'data', []),
+  const flatten = ({ clusterServiceVersions, subscriptions, allClusterServiceVersions }) => {
+    const csvData = _.get(clusterServiceVersions, 'data', []);
+    const allCSVFiltered = _.get(allClusterServiceVersions, 'data', []).filter((el) => {
```

Comment:
Use optional chaining.

Reply:
@zherman0 can you explain a bit why we need this extra logic and what it's doing?

Reply:
As mentioned above, I am now combining the gloabl CSVs and the namespaced CSVs into one object so I can show more timely information to the user.  There are issues with this for non-admin users so this needs to be reworked.